.htaccess

```diff
@@ -29,6 +29,16 @@
     RewriteRule ^storage/framework/.* index.php [L,NC]
     RewriteRule ^storage/temp/protected/.* index.php [L,NC]
     RewriteRule ^storage/app/uploads/protected/.* index.php [L,NC]
+    
+    ##
+    ## Disable PHP in storage folder
+    ##
+    RewriteRule ^storage/.*\.(?:php[1-6]?)$ - [NC,F]
```

Comment:
This shouldn't be needed since all PHP files should be blocked from running except for index.php, and those files should not be allowed to be uploaded in the first place.

```
 ##
    ## Block all PHP files, except index
    ##
    RewriteCond %{REQUEST_FILENAME} -f
    RewriteCond %{REQUEST_FILENAME} \.php$
    RewriteRule !^index.php index.php [L,NC]
```

Reply:
Done